### DISCUSSION TITLE
Migrating from Kalendae

### DISCUSSION THREAD

**Discussion by stell** [easepick/easepick#72] [Q&A]
Easepick has a great range selector and a great grid feature.
Can someone tell if moving from Kalendae if causing some major missing parts?

https://github.com/Twipped/Kalendae

**Comment by wakirin** [comment]
I haven't used the Kalendae library before, so I can't say about missing parts.

If you already tried to migrate you can show your current config Kalendae and what you have tried to do (jsfiddle).

**Comment by stell** [comment]
The config is rather huge, but here is an image and if someone wants to play around with it:
https://www.bodenseeurlaub.de/en/search/51

![kalendae](https://user-images.githubusercontent.com/701298/171849195-213d3baa-d6a6-4225-bd8e-c71896c13f21.png)
